ci/generate-markdown/generate-readme.sh

```diff
@@ -36,7 +36,7 @@ file="README.md"
 (
   mapfile -t libraries < <(bazelisk --batch query \
     --noshow_progress --noshow_loading_progress \
-    'kind(cc_library, //:all) except filter("experimental|mocks|common|grpc_utils|internal", kind(cc_library, //:all))' |
+    'kind(cc_library, //:all) except filter("experimental|mocks|common|grpc_utils", kind(cc_library, //:all))' |
```

Comment:
is this change related?

Reply:
Tangentially, at least.  While investigating the _"common is not a valid feature in google-cloud-cpp"_ CMake warning elicited through similar code in check-api.sh, I noticed that there isn't a `//:*internal*` target, so there's no reason to try to filter it out.  So I slipped in this update. :-)